### PR TITLE
[ADMIN] [s] add pretty filters to announcement & shuttlecall, cd to alert level set

### PR DESCRIPTION
**Pull request by ynot01**
IMPORTANT_ACTION_COOLDOWN on setting security alert level

pretty filter on silicon announce

pretty filter on shuttle call reason

:cl:
tweak: Security alert level now has a 10 second cooldown
/:cl:

### CODE REVIEW

**Comment by adamsong** [comment]
Changelog

**Comment by ToasterBiome** [comment]
Can you test this on a local server?

File: code/game/machinery/computer/communications.dm [hunk 1]
```diff
@@ -41,6 +41,9 @@
 	/// The last lines used for changing the status display
 	var/static/last_status_display
 
+	/// The next time the security level can be changed from this console
+	var/next_lvl_change = 0
```
Comment (Bibby0110) [inline code/game/machinery/computer/communications.dm]: Use proper cooldown defines

**Review by Bibby0110 — DISMISSED** [review]

File: code/game/machinery/computer/communications.dm (hunk 1 above)
Comment (ynot01) [reply code/game/machinery/computer/communications.dm]: 0 SECONDS?

Comment (Bibby0110) [reply code/game/machinery/computer/communications.dm]: no, search for COOLDOWN_DECLARE and such

Comment (ynot01) [reply code/game/machinery/computer/communications.dm]: oh wow thats neat

Comment (ynot01) [reply code/game/machinery/computer/communications.dm]: im an idiot theres already a thing here

File: code/game/machinery/computer/communications.dm [hunk 2]
```diff
@@ -126,6 +126,10 @@
 			if (!authenticated_as_silicon_or_captain(usr))
 				return
 
+			if (!COOLDOWN_FINISHED(src, important_action_cooldown))
+				return
+
+
```
Comment (JamieD1) [inline code/game/machinery/computer/communications.dm]:
```suggestion
			if (!COOLDOWN_FINISHED(src, important_action_cooldown))
				return
```

File: code/game/machinery/computer/communications.dm [hunk 3]
```diff
@@ -156,6 +160,9 @@
 			deadchat_broadcast(" has changed the security level to [params["newSecurityLevel"]] with [src] at [span_name("[get_area_name(usr, TRUE)]")].", span_name("[usr.real_name]"), usr)
 
 			alert_level_tick += 1
+
+			COOLDOWN_START(src, important_action_cooldown, IMPORTANT_ACTION_COOLDOWN)
+
```
Comment (JamieD1) [inline code/game/machinery/computer/communications.dm]:
```suggestion
			COOLDOWN_START(src, important_action_cooldown, IMPORTANT_ACTION_COOLDOWN)
```

**Review by JamieD1 — CHANGES_REQUESTED** [review]

File: code/game/machinery/computer/communications.dm [hunk 4]
```diff
@@ -126,6 +126,10 @@
 			if (!authenticated_as_silicon_or_captain(usr))
 				return
 
+			if (!COOLDOWN_FINISHED(src, important_action_cooldown))
+				return
+
+
 			// Check if they have
```
Comment (JamieD1) [inline code/game/machinery/computer/communications.dm]:
```suggestion
			if (!COOLDOWN_FINISHED(src, important_action_cooldown))
				return

			// Check if they have
```

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**